db/migrate/20150126160624_create_notifications.rb

```diff
@@ -0,0 +1,15 @@
+class CreateNotifications < ActiveRecord::Migration
+  def change
+    create_table :notifications do |t|
+      t.references :article, index: true
```

Comment:
do you need to reference the article from notifications? isn't it redundant if the comment already references the article?